src/components/Dissolution/AssociationDetails.vue

```diff
@@ -47,13 +46,28 @@
           <div v-else>Same as Mailing Address</div>
         </v-col>
       </v-row>
-    </div>
+    </article>
+
+    <template v-if="showBusinessDate">
+          <v-divider class="mx-6" />
```

Comment:
Fix indentation.

Also add a blank line after this (consistent with other HTML in here).